### PR TITLE
Deprecated: (2)Remove OASIS xsl stylesheets: The goal of xsl/ structure. 

### PR DESCRIPTION
**Pull request by usamimikamen-sk**
Depends on PR #229
----
Reorereing PR #211 - #224 and https://github.com/usamimikamen-sk/gnucash-docs/issues/3#issue-1006428027. 
The start is b6ff6a4.

```
cmake -S ../gnucash-docs -B . -DWITH_PDF=OFF -DWITH_EPUB=OFF
cmake --build . --target html
cmake --install . --prefix ${prefix}
```

Then ghelp and HTML will be built and installed successfully.
Customization of XSL files is omitted. Therefore simple HTML files
will be created.

### CODE REVIEW

File: cmake/AddGncDocTargets.cmake [hunk 1]
```diff
@@ -1,49 +1,66 @@
...
+#
 function (add_gnc_doc_targets docname entities)
 
     get_filename_component(lang ${CMAKE_CURRENT_SOURCE_DIR} NAME)
 
+    # Both PNG and SVG are used as figures. Some files are installed
+    # but may not be used.
     file(GLOB_RECURSE figures
         "${CMAKE_CURRENT_SOURCE_DIR}/figures/*.png"
         "${CMAKE_CURRENT_SOURCE_DIR}/figures/figures/*.svg")
```
Comment (usamimikamen-sk) [inline cmake/AddGncDocTargets.cmake:21]: It should be fixed another commit.

**Comment by jralls** [comment]
I'm concerned about all of the commit messages that start with "Caution: This commit leads to build failure." That's a big red flag that you haven't sequenced the work correctly. One way to maintain successful builds throughout is adjusting the CMakeLists.txt files so that they build both with and without the OASIS stylesheets in the sources, then remove the OASIS stylesheets, then remove from the CMakeLists.txt files the ability to use them.

**Comment by usamimikamen-sk** [comment]
It is not easy because there are many requests and constraints.
Please give me time to think about the order. 

(1) request and constraints
* Remove OASIS XSLT from xsl/1.79.2 and use system xsl files.
* Move contents of stylesheet/ to xsl/.
* GnuCash-specific icons are wanted to be used. (by @fellen)
* Create dependency for GnuCash-specific icons. (by @gjanssens)
* Create dependency for GnuCash-customized xsl files. (by @gjanssens)
* Don't break the build process for each commit. (by @jralls and @fellen)
* Avoid regression. (by all people)

(2) Current bugs and regressions
* Paper size for PDF is set USletter (except LANG C.) (cmake regression)
* TOC of chm for ja and ru is not shown correctly. (cmake regression)
* Dependency for language-specific DTD files is missing. (new by PR #207)
* Current variablelist.xsl breaks docbook-xsl multiple <term> feature, should be split for HTML and FO.
* Customization of titlepage is not applied, should be split for HTML and FO.


(3) new customization
* Customize variable lists, PR #203. (by @CWehli)
* Set spacing="compact" for HTML/PDF. (by @CWehil)
* Add clean-extra target. (by @usamimikamen-sk)
* Change install directory for ghelp. (by @fellen and @gjanssens)
* Underline link in PDF files. (by @CWehil)
* Update Japanese translation for GnuCash-Guide. (by @usamimikamen-sk)
* Create Japanese translation for GnuCash-Help. (by @usamimikamen-sk)
* Move appendix aux file format from GnuCash-Guide to GnuCash-Help. (by @fellen)

If you find other requests or bugs, please tell me.

**Comment by fellen** [comment]
> It is not easy because there are many requests and constraints.

It is not quite clear from your description: Which are old bugs in Cmake and which are a follow up of your main change? Perhaps split them also into separate commits before and after your main change.

Off topic: Perhaps you can save work and time by restricting your PRs on

- one _structural_ change
 and use the remaining time on

- _file local_ content improvements like <q>Review chapter X of guide/C</q>
until the structural change is accepted/merged.
